props/saber_fett263_buttons.h

```diff
@@ -2346,8 +2350,12 @@ SaberFett263Buttons() : PropBase() {}
       StopTrackPlayer();
     }
     if (!SFX_medit) {
+#ifdef DISABLE_TALKIE
```

Comment:
This should call errors::error_in_font_directory() instead.

Reply:
updated